src/config/MetaSchema.json

```diff
@@ -65,19 +81,195 @@
                 },
                 "script": {
                     "type": "string"
+                },
+                "range": {
+                    "type": "object",
+                    "oneOf": [
+                        {
+                            "required": [
+                                "min"
+                            ],
+                            "properties": {
+                                "min": {
+                                    "type": "integer"
+                                },
+                                "exclusiveMin": {
+                                    "not": {}
```

Comment:
maybe i don't need the not if I use `additionalProperties : false`, will test that